flexmeasures/cli/data_add.py

```diff
@@ -1638,6 +1215,22 @@ def add_schedule(  # noqa C901
             click.secho("New schedule is stored.", **MsgStyle.SUCCESS)
 
 
+@fm_add_data.group(
```

Comment:
Putting this here overwrites the new `flexmeasures add schedule`, so it doesn't work anymore.

So my current recommendation is to remove it, and let people know in the changelogs (main one and also the CLI one) that we replaced the two specific commands with a general one.

I asked Felix if he wants to veto that.